README.md

```diff
@@ -1,32 +1,92 @@
 # Arizona Icons
+Find more detailed information on how to use `az-icons` within the (Arizona
+Bootstrap Docs)[https://digital.arizona.edu/arizona-bootstrap/docs/2.0/icons/].
+## Download
+Download Arizona Icons to get the compiled CSS and font files, source code, or
+include it with your favorite package managers like npm, RubyGems, and more.
 
-## Using the Icons
-The icons can be used with Arizona Digital CDN link:
+### Source Files
+Compile Arizona Icons with your own asset pipeline by downloading our source
+CSS, SVG, PNG, and font files. This option requires some additional tooling:
 
-```html
-<link href="https://cdn.digital.arizona.edu/lib/az-icons/main/az-icons-styles.css" rel="stylesheet">
+### Arizona Digital CDN
+Skip the download with the Arizona Digital CDN to deliver a cached version of
+Arizona Icons’s CSS and font files to your project.
+
+When choosing which version to include in your project, take these three choices
+into account:
+
+**Stable** version: Each tagged release has a permanent CDN link. This version
+has to be upgraded on your project manually.
+```
+<link rel="stylesheet" href="https://cdn.digital.arizona.edu/lib/az-icons/1.0.0/az-icons-styles.css" crossorigin="anonymous">
+```
+
+**Latest** tagged version: The contents of this CDN link update automatically
+every time we tag a release of `az-icons`. While there is some risk involved
+with this link, for example, the icons could randomly break on your site, the
+up-side is that you get access to new icons and upgrades to performance on each
+release automatically. It's probably OK to use on production sites. Please
+report bugs to the AZ Digital team.
```

Comment:
Would it be good to link "report bugs" to the Issues page or another page? The same report bugs link could be added to the Main version text below, or perhaps that note could be moved elsewhere.